diff --git a/opencensus/exporters/trace/stackdriver/BUILD b/opencensus/exporters/trace/stackdriver/BUILD
index aa45c01..3335e9b 100644
--- a/opencensus/exporters/trace/stackdriver/BUILD
+++ b/opencensus/exporters/trace/stackdriver/BUILD
@@ -37,7 +37,6 @@ cc_library(
         "@com_google_absl//absl/memory",
         "@com_google_absl//absl/strings",
         "@com_google_absl//absl/time",
-        "@com_google_googleapis//google/devtools/cloudtrace/v2:trace_cc_grpc",
-        "@com_google_googleapis//google/devtools/cloudtrace/v2:trace_cc_proto",
+        "@googleapis//:tracing_proto",
     ],
 )
